molecule/__init__.py

```diff
@@ -18,8 +18,21 @@
 #  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 #  THE SOFTWARE.
 
+from signal import SIGINT
+from signal import signal
+from sys import exit
```

Comment:
One small nit.  Third party code in molecule was refactored to `import library`.  Can you follow that?

```
import signal
import sys
```